plugins/wallet_plugin/include/eosio/wallet_plugin/wallet_manager.hpp

```diff
@@ -126,7 +133,7 @@ class wallet_manager {
    std::chrono::seconds timeout = std::chrono::seconds::max(); ///< how long to wait before calling lock_all()
    mutable timepoint_t timeout_time = timepoint_t::max(); ///< when to call lock_all()
    boost::filesystem::path dir = ".";
-   std::string eosio_key = "5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3";
+   std::string eosio_key;
```

Comment:
This can be removed as not used now.

Reply:
You can still pass it on the command line as an option.

Reply:
Please remove that feature as well.